helm/vault-exporter/values.yaml

```diff
@@ -0,0 +1,17 @@
+name: vault-exporter
+namespace: vault-exporter
```

Comment:
I would prefer those values to be aligned with that https://github.com/giantswarm/aws-operator/blob/40cbaaff12dab82ac4d0283024407172376da14e/helm/aws-operator/values.yaml#L31-L36 but I won't block because of that.